### PR TITLE
fix for authentication problem

### PR DESCRIPTION
**Pull request by kdeyev**

### CODE REVIEW

**Comment by kdeyev** [comment]
It addresses the #144

File: smart_meter_texas/const.py [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-AUTH_ENDPOINT = "/user/authenticate"
-DASHBOARD_ENDPOINT = "/dashboard"
-LATEST_OD_READ_ENDPOINT = "/usage/latestodrread"
-METER_ENDPOINT = "/meter"
-OD_READ_ENDPOINT = "/ondemandread"
-INTERVAL_SYNCH = "/adhoc/intervalsynch"
+BASE_HOSTNAME = "www.smartmetertexas.com"
+BASE_URL = "https://" + BASE_HOSTNAME
+BASE_ENDPOINT = BASE_URL 
+AUTH_ENDPOINT = "/commonapi/user/authenticate"
```
Comment (nschimme) [inline smart_meter_texas/const.py:6]: Why change so much of the logic as part of this diff?

We could instead do here:
`AUTH_ENDPOINT = BASE_URL + "/commonapi/user/authenticate"`

That way everything else would be untouched.

File: smart_meter_texas/const.py [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
-    "Content-Type": "application/json",
-    "Accept": "application/json",
+    "accept": "application/json, text/plain, */*",
+    "accept-language": "en-US,en;q=0.9",
+    "cache-control": "no-cache",
+    "content-type": "application/json",
+    "origin": "https://www.smartmetertexas.com",
+    "pragma": "no-cache",
+    "referer": "https://www.smartmetertexas.com/home",
+    "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36",
```
Comment (nschimme) [inline smart_meter_texas/const.py:25]: Shouldn't we be using the `USER_AGENT_TEMPLATE` here?

File: smart_meter_texas/const.py (hunk 1 above)
Comment (kdeyev) [reply smart_meter_texas/const.py]: Makes sense

File: smart_meter_texas/const.py (hunk 2 above)
Comment (kdeyev) [reply smart_meter_texas/const.py:25]: yes, it was redundant

Comment (kdeyev) [reply smart_meter_texas/const.py]: Done. I had to update the chrome version

**Review by nschimme — APPROVED** [review]
LGTM but I'm just doing this to help speed it up for @grahamwetzler . We still need to wait for him to merge.

**Comment by grahamwetzler** [comment]
Thanks @kdeyev and @nschimme! 🎉

**Pull request merged**